src/compiler/ast/function/ArrowFunction.ts

```diff
@@ -13,4 +12,30 @@ export class ArrowFunction extends ArrowFunctionBase<ts.ArrowFunction> {
     getEqualsGreaterThan(): Node<ts.Token<SyntaxKind.EqualsGreaterThanToken>> {
         return this.getNodeFromCompilerNode(this.compilerNode.equalsGreaterThanToken);
     }
+
+    /**
+     * Add braces to this arrow function body in case is viable.
+     *
+     * WARNING! This will forget all the nodes in the file! It's best to do this after you're all done with the file.
```

Comment:
Could you remove this from the PR? There are some refactors implemented in `getEditsForRefactor` that we can do a better job at that won't cause forgetting all the nodes.